test/main.cpp

```diff
@@ -3,12 +3,27 @@
 #include <rs/utils/common.h>
 #include "rs/flowcontrol/RSAnalysisEngineManager.h"
 #include "rs/flowcontrol/RSAnalysisEngine.h"
+#include <string>
+#include <sys/stat.h>
+#include <stdio.h>
+#include <gtest/gtest.h>
+#include "rs/flowcontrol/RSAnalysisEngineManager.h"
+#include "rs/flowcontrol/RSPipelineManager.h"
+
+#include "rs/flowcontrol/RSAnalysisEngine.h"
+#include <rs/utils/common.h>
+#include <rs/types/all_types.h>
+#include <rs/scene_cas.h>
+
+#include <pcl/point_types.h>
+#include <ros/ros.h>
 
+#include <mongo/client/dbclient.h>
 
+//Variables to be tested
 
-int main(int argc, char **argv)
+		
+int main2(int argc, char **argv)
```

Comment:
tis looks weird? why main2?

Reply:
Reviewed for the unit test purposes, we used the main method into the unit test